### PR TITLE
toggle help on kebab click

### PR DESCRIPTION
**Pull request by edewit**
fixes: #305

### CODE REVIEW

File: src/PageHeader.tsx [hunk 1]
```diff
@@ -75,7 +75,12 @@ export const Header = () => {
 
   const HelpDropdownItem = () => {
     const { t } = useTranslation();
-    return <DropdownItem icon={<HelpIcon />}>{t("help")}</DropdownItem>;
+    const { toggleHelp } = useContext(HelpContext);
+    return (
+      <DropdownItem icon={<HelpIcon />} onClick={toggleHelp}>
```
Comment (jenny-s51) [inline src/PageHeader.tsx:80]: Would be helpful to have some sort of indicator here when help is toggled... maybe a checkbox or a switch. There is currently no way of telling that help is on or off unless you specifically check for changes on the page

**Review by jenny-s51 — CHANGES_REQUESTED** [review]
Toggle functionality working as expected - see comment below

**Review by ssilvert — APPROVED** [review]
LGTM

File: src/PageHeader.tsx [hunk 2]
```diff
@@ -75,7 +75,12 @@ export const Header = () => {
 
   const HelpDropdownItem = () => {
     const { t } = useTranslation();
-    return <DropdownItem icon={<HelpIcon />}>{t("help")}</DropdownItem>;
+    const { enabled, toggleHelp } = useContext(HelpContext);
+    return (
+      <DropdownItem icon={<HelpIcon />} onClick={toggleHelp}>
+        {t("help")} {enabled ? t("on") : t("off")}
```
Comment (ssilvert) [inline src/PageHeader.tsx:81]: This is not always localizable.  You need to translate as a single phrase.

**Review by ssilvert — CHANGES_REQUESTED** [review]
Noticed something here.

**Review by jenny-s51 — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**